applications/launchpad/docker_rig/Readme.txt

```diff
@@ -0,0 +1,148 @@
+# [WIP] Docker deploy of TariLabs Suite
+
+copy env-sample-local-build to .env and update as needed
+
+# Tari Network
+TARI_NETWORK=dibbler
+
+# Docker data folder
+DATA_FOLDER=tmp/dibbler
+
+# Enable wait for tor network (only basic sleep)
+WAIT_FOR_TOR=1
+
+# For local image pulling and developement, set the following envs as follows - default is quay.io/tarilabs - latest images
+TL_TAG_URL=local-tarilabs
+
+```
+docker-compose build
+```
+
+# Monero Envs
+TARI_MONEROD_USERNAME=tari
+TARI_MONEROD_PASSWORD=tari
+# Leave blank for mainnet
+MONEROD_EXTRA_OPTIONS="--testnet"
+
+First run to create base_node ids
+```
+docker-compose run --rm base_node --create-id --init
+```
+
+Should then be able to do a complete start up
+```
+docker-compose up -d
+```
+
+Check the tor is running
+```
+docker-compose logs tor
+```
+
+Check that base node is running
+```
+docker-compose logs base_node
+```
+
+Might be a chance that base_node id were not create or the first step missed, and there is a problem like
+```
+Y/n
+```
+In this case, stop the base_node with 
+```
+docker-compose stop base_node
+docker-compose run --rm base_node --create-id --init
```

Comment:
```suggestion
docker-compose run --rm base_node --init
```

Reply:
This is first run, before there is an ID. base_node will ask to create ID and wait forever for console input. I will test, but the above is note for init run only.

Reply:
Updated tag to include arch and feature.

Removed latest linking and pushing.

Reply:
Wondering if the tag should not also include the git commit hash?